docs/faq.md

```diff
@@ -0,0 +1,89 @@
+---
+Title: Frequently Asked Questions
+---
+
+## General Questions
+
+### Q: What is Headlamp and who is it for?
+
+**A:** Headlamp is an open source, extensible graphical user interface (GUI) designed to simplify the management of Kubernetes clusters. It serves as a visual management tool ideal for developers, DevOps professionals, system administrators, and overall Kubernetes users, who are looking for an intuitive way to oversee cluster operations, deployments, and troubleshooting.

```

Comment:
This is a bit complex to read.

![image](https://github.com/headlamp-k8s/headlamp/assets/9541/d07d5a6d-9563-4445-b68e-5bed7fb29707)

Reply:
There's many complex sentences which are hard for people to read.

![image](https://github.com/headlamp-k8s/headlamp/assets/9541/0596ca42-ca63-49b3-a9cf-f080a19bd3a1)

Reply:
It doesn't want to be flexible apart from this... I think this may be a push for the paid features..

adding the change it allows.

![image](https://github.com/headlamp-k8s/headlamp/assets/78232183/2e185dee-71da-4797-860e-cbb5ad900a8a)